ts/features/walletV3/payment/screens/WalletPaymentCheckoutMethodSelectionScreen.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  ButtonSolid,
+  GradientScrollView,
+  H2,
+  H6,
+  HeaderSecondLevel,
+  IOLogoPaymentType,
+  RadioGroup,
+  RadioItem,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { useNavigation } from "@react-navigation/native";
+import * as React from "react";
+import { View } from "react-native";
+import TypedI18n from "../../../../i18n";
+
+const ListItemHeading = (props: { text: string }) => (
+  // FIXME:: needs new DS component
+  <View style={{ paddingVertical: 16 }}>
+    <H6>{props.text}</H6>
+  </View>
+);
+
+type PaymenMethod = {
+  icon: IOLogoPaymentType;
+  name: string;
+};
+
+const savedMethods: Array<PaymenMethod> = [
+  {
+    icon: "amex",
+    name: "amex ***1"
+  },
+  {
+    icon: "amex",
+    name: "amex ***2e"
+  },
+  {
+    icon: "amex",
+    name: "amex ***3"
+  }
+];
+
+const notSavedMethods: Array<PaymenMethod> = [
+  {
+    icon: "amex",
+    name: "amex ***4"
+  },
+  {
+    icon: "amex",
+    name: "amex ***5"
+  },
+  {
+    icon: "amex",
+    name: "amex ***6"
+  }
+];
```

Comment:
We should remark that these types are not final and only mock data. What do you think?